### ISSUE TITLE
"white noise" when click play

### ISSUE THREAD

**Issue by nicbre85** [DeaDBeeF-Player/deadbeef#2886]
I've just updated the software (1.9.3) and everytime i play a song all i can hear is a very loud "white noise". 
Then, if i click on another track the noise disappears.

Deadbeef version: 1.9.3
OS: archlinux

**Comment by nicbre85** [comment]
update: i've downgraded pipewire to the prev version, now deadbeef works fine, so i guess it's a pipewire issue

**Comment by Oleksiy-Yakovenko** [comment]
Deadbeef doesn't (yet) support pipewire natively out of the box, so at the very least you need to install the 3rd party pipewire plugin.
Either way, regardless of what the problem is -- you have the following options to try.
1. don't use pipewire
2. use the 3rd party pipewire plugin for deadbeef, [from here](https://github.com/saivert/ddb_output_pw)
3. wait until deadbeef supports pipewire natively, although this would be the same as [2]
4. If none of the above works -- you should probably consider seeking help with pipewire devs, or the pipewire plugin devs.

**Comment by DragoonAethis** [comment]
I also have the same issue with the ALSA output while using PipeWire (output device is explicitly set to "PipeWire Sound Server", and that goes into the default output which is a USB DAC). Installing the PipeWire output plugin fixed the thing, but I tried to check what's going on.

Diffing pw-dump outputs before/after retrying shows one meaningful difference:
- File 1: `Stream #0:0: Audio: opus, 48000 Hz, stereo, fltp` - first playback attempt sets deadbeef's output format to S16LE 44100 (noise), retrying sets F32LE 48000 (which is fine).
- File 2:`Stream #0:0: Audio: mp3, 44100 Hz, stereo, fltp, 128 kb/s` - first playback attempt sets S16LE 44100 (noise), but F32LE 44100 after retrying.

Looks like a regression in PW 0.3.60. On Arch, downgrading `pipewire-audio` to 0.3.59 solves the issue (core, ALSA, etc can stay at 0.3.60). I've submitted an issue there: https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/2832
